utils/python/CIME/utils.py

```diff
@@ -872,3 +873,49 @@ def touch(fname):
         os.utime(fname, None)
     else:
         open(fname, 'a').close()
+
+def find_system_test(testname, case):
+    '''
+    Find and import the test matching testname
+    Look through the paths set in config_files.xml variable SYSTEM_TESTS_DIR
+    for components used in this case to find a test matching testname.  Add the
+    path to that directory to sys.path if its not there and return the test object
+    Fail if the test is not found in any of the paths.
+    '''
+    system_test_path = None
+    if testname in ("TESTRUNPASS", "TESTRUNDIFF", "TESTRUNFAIL", "TESTBUILDFAIL",
```

Comment:
There might be a more elegant way of doing this. I'd say if the testname matches anything in the system_tests_common, then we use that. That would allow us to avoid having to maintain this list.